### ISSUE TITLE
Code Commenting

### ISSUE THREAD

**Issue by cjhetzle** [SirQuacker/ROR2_Mod_Trick#1]
All of the code's comments have been either removed for production or were never there (prob the first) and we need to be able to more easily work through the code and maybe create documentation to navigate through the work.

Please, take some time and read through the code and write comments for what everything does in a neat and orderly manor, and if you are not primarily working on commenting, please just comment as you are working on something.

**Comment by Anti-Antidote** [comment]
I would like to help with this, would you make me a contributor so I can get comments in? If not I can just PR stuff.

**Comment by Anti-Antidote** [comment]
Also, are we following standard [C# XML documentation](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/xml-documentation-comments) or just documenting how we want? It might be less messy if we stick to a guide.

**Comment by cjhetzle** [comment]
Yeah we should stick to something. I would say yes, go with the XML Doc

**Comment by SirQuacker** [comment]
this is outdated i have a new project that is compatible with the current mod loader
